frontend/tests/badges/cppCourseGraduateTest.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+/**
+ * Simple test for legacy user Badge
+ *
+ * @author RuizYugen
+ */
+class CppCourseGraduate extends \OmegaUp\Test\BadgesTestCase {
+    public function testCourseCppUserWinBadge() {
```

Comment:
nit:

```suggestion
    public function testCourseCppUserEarnBadge() {
```

igual con la otra.